components/Hero.js

```diff
@@ -3,18 +3,31 @@ import Image from 'next/image';
 import styles from '../components/hero.module.css';
 import utilsStyles from '../styles/utils.module.css';
 import Menu from './menu';
+import { useMediaQuery } from 'react-responsive';
 
 export default function Hero(){
+
+  const isDesktop = useMediaQuery({
+    query: '(min-width: 1024px)'
+  })
+
+  const isTablet = useMediaQuery({
+    query: '(min-width: 768px)'
+  })
+
   return (
 		<div className={utilsStyles.bg_black}>
         <Menu />
         <div className={styles.logo_wrap}>
           <Image className={styles.logo} width={92} height={120} src="/assets/SeattleDJ_fist.png" alt="logo"/>
         </div>
         <div className={styles.top}>
-          <h1 className={`${utilsStyles.font_white} ${styles.title}`}>Protest<br/><span className={utilsStyles.font_accent}>T</span>unes</h1>
+        {isDesktop ? <h1 className={`${utilsStyles.font_white} ${styles.title}`}>Protest <span className={utilsStyles.font_accent}>T</span>unes</h1> 
+        : isTablet ? <h1 className={`${utilsStyles.font_white} ${styles.title}`}>Protest <span className={utilsStyles.font_accent}>T</span>unes</h1>
+        : <h1 className={`${utilsStyles.font_white} ${styles.title}`}>Protest<br/><span className={utilsStyles.font_accent}>T</span>unes</h1>}
         </div>
-        <p className={styles.intro}>Musician, Singer, Rapper. He just debuted as Music Artist to follow his dream. Now it’s not ‘dream’ anymore.</p>
+        {isDesktop ? <h2 className={styles.intro}>Musician, Singer, Rapper. He just debuted as Music Artist to follow his dream. Now it’s not ‘dream’ anymore.</h2>
+        : <h2 className={styles.intro}>Musician, Singer, Rapper. He just debuted as Music Artist to follow his dream.<br/> Now it’s not ‘dream’ anymore.</h2>}
```

Comment:
```suggestion
        {isDesktop ? <p className={styles.intro}>Musician, Singer, Rapper. He just debuted as Music Artist to follow his dream. Now it’s not ‘dream’ anymore.</p>
        : <p className={styles.intro}>Musician, Singer, Rapper. He just debuted as Music Artist to follow his dream.<br/> Now it’s not ‘dream’ anymore.</p>}
```
@Shiho317 This is more like a paragraph, so let's use `<p>` tag instead. Also, its font family was different from the design.

Plus, can we put this paragraph inside this div `<div className={styles.top}>` ? It looks more natural from html structure perspective.